### DISCUSSION TITLE
PDFBOX timeline

### DISCUSSION THREAD

**Discussion by naintrakusam** [TomRoush/PdfBox-Android#323] [General]
Hi, I would like to ask you when you expect this library to be at version 2.0.16 ?

_"Hi, as stated in pdfbox documentation, to ensure that annotations will be visible in all viewers, one should call this after adding annotation to the page's annotation list, before saving the document :

**annotations.forEach(ann -> ann.constructAppearances(document));**

But in current release, constructAppearances is still missing. Do you think it can be added in near feature ?"_

"Since 2.0.16 the appearance stream of many annotation types can be created with annotation.constructAppearances(). –
Tilman Hausherr
Aug 13 '19 at 9:32"

Thank you for response

**Comment by TomRoush** [comment]
I added milestones for all the 2.0.X API version releases with the estimated release dates : [Milestones](https://github.com/TomRoush/PdfBox-Android/milestones) The current schedule is a new release every other Sunday, but that's subject to change depending on how long they take. Based on that the estimated date for 2.0.16.0 would be March 6th. I'll follow up more on your specific request in the issue you created
